src/axolotl/utils/vessl_.py

```diff
@@ -0,0 +1,14 @@
+"""Module for vessl utilities"""
+
+import os
+
+from axolotl.utils.dict import DictDefault
+
+
+def setup_vessl_env_vars(cfg: DictDefault):
+    # VESSL_RUN_INITIAL_CONFIG is a variable that contain path to
+    # default credential inside a VESSL Run
+    credential_path = os.environ.get("VESSL_RUN_INITIAL_CONFIG")
+    if credential_path:
+        cfg.use_vessl = True
```

Comment:
In what case `cfg.vessl_credential_path` is not `None` and `cfg.use_vessl` is `False`?

Reply:
Both are `None` by default if not set to a value. I will check if adding the parameter to training yaml file may cause them to have a value.

Reply:
1. if you want to add configs, you need to touch this file: https://github.com/Y-IAB/axolotl/blob/main/src/axolotl/utils/config/models/input/v0_4_1/__init__.py
2. do we need both values? if cfg.vessl_credential_path is set, can't we consider it that vessl is enabled?

Reply:
1. No, I don't want to add config. I just want it to be enabled automatically inside a VESSL Run.
2. True, I will update it.

Reply:
but you are using `cfg` here. for type checking, I think you need to add one